MAINTAINERS

```diff
@@ -0,0 +1,2 @@
+Andreas Fuchs <andreas.fuchs@sit.fraunhofer.de>
```

Comment:
Are you planning on signing the tags/release tarballs using PGP like the other tpm2-software projects do? If so, the public keys should be uploaded to a key server and the process should be documented in a `RELEASE.md` file [like tpm2-tss does](https://github.com/tpm2-software/tpm2-tss/blob/master/RELEASE.md) (which might be nice to have regardless).

Reply:
What are typical key servers you see as maintainer ? I have it uploaded to github already, so there is at least this "green batch" available.
You get to choose one...

Reply:
RELEASE.md was added.
@diabonas looking reasonable ?

Reply:
RELEASE.md looks good!

I don't think you can download the associated PGP key from GitHub, but I am not quite sure. I would recommend uploading the key to one of the keyservers in the SKS pool: http://pool.sks-keyservers.net:11371/ These servers are all synchronized, so uploading the key to any one of them is enough. Since many GnuPG installations have one of the SKS servers as a default keyserver, `gpg --receive-keys` should work out of the box for most people.

Reply:
Done